lib/modules/platform/index.ts

```diff
@@ -59,6 +59,12 @@ export async function initPlatform(config: AllConfig): Promise<AllConfig> {
     // TODO: null check (#7154)
     matchHost: URL.parse(returnConfig.endpoint).hostname!,
   };
+  // There might have been platform-specific modifications to the token
+  if (returnConfig.token) {
+    config.token = returnConfig.token;
+    platformRule['token'] = returnConfig['token'];
+    delete returnConfig['token'];
```

Comment:
Won't these two lines be performed below on lines 73/74, meaning they can be removed/

Reply:
Good catch! Pushed a new commit